src/chigame/api/tests/factories.py

```diff
@@ -106,3 +106,18 @@ class Meta:
         model = Chat
 
     tournament = SubFactory(TournamentFactory)
+
+
+class LobbyFactory(DjangoModelFactory):
+    class Meta:
+        model = Lobby
+
+    match_status = Lobby.Lobbied
+    name = Faker("word")
+    game = SubFactory(GameFactory)
+    game_mod_status = Lobby.Default_game
+    created_by = SubFactory(UserFactory)
+    min_players = 2
```

Comment:
I'm not fully convinced that we have set values for these attributes. What if we wanted to test the creation of other factories that don't fit into this. An alternate could be employing lazy attributes as below:

```
min_players = factory.LazyAttribute(lambda x: random.randint(2, 6))
```